evm/src/cpu/columns/general.rs

```diff
@@ -144,5 +155,15 @@ pub(crate) struct CpuJumpsView<T: Copy> {
     pub(crate) should_trap: T,
 }
 
+#[derive(Copy, Clone)]
+pub(crate) struct CpuShiftView<T: Copy> {
+    // 1 if the high limbs of the displacement are all zero, 0 if one
+    // or more of the limbs is nonzero.
+    pub(crate) displacement_high_limbs_are_zero: T,
```

Comment:
This is always equal to `two_exp.used`, so you could just use that and not need the equality constraint